sriovnet.go

```diff
@@ -382,3 +383,23 @@ func FreeVfByNetdevName(handle *PfNetdevHandle, vfIndex int) error {
 func GetVfNetdevName(handle *PfNetdevHandle, vf *VfObj) string {
 	return vfNetdevNameFromParent(handle.PfNetdevName, vf.Index)
 }
+
+// GetVfIndexByPciAddress gets a VF PCI address (e.g '0000:03:00.4') and
+// returns the correlate VF index.
+func GetVfIndexByPciAddress(vfPciAddress string) (string, error) {
```

Comment:
@moshe010 In previously merged API GetVfPresentor() accepts vfIndex as int.
Here returned vIndex is string. Lets have it consistent to have index as 'int' or uint32.